appinventor/components/src/com/google/appinventor/components/runtime/ChartModel.java

```diff
@@ -53,7 +51,62 @@ public void setLabel(String text) {
     /**
      * Sets the elements of the Data Series from a CSV-formatted String.
      *
+     * TODO: Optimization step: 1.Add values to list 2.Sort values 3.Set values to dataset
```

Comment:
Methods for both list import and set elements can be optimized

Reply:
Add an issue for this instead.

For now, this should not be done, since not all Data Series might have to be sorted.

Reply:
Covered in #67